socorro/processor/processor_pipeline.py

```diff
@@ -201,11 +174,71 @@ def __init__(self, config, rules=None, quit_check_callback=None):
         else:
             self.quit_check = lambda: False
 
-        rule_set = rules or list(DEFAULT_RULES)
+        self.rules = rules or self.get_ruleset(config)
+        for rule in self.rules:
+            self.logger.info('Loaded rule: %r' % rule)
+
+    def get_ruleset(self, config):
+        """Generate rule set for Mozilla crash processing.
+
+        :arg config: configman DotDict config instance
 
-        self.rules = []
-        for a_rule_class in rule_set:
-            self.rules.append(a_rule_class(config))
+        :returns: pipeline of rules
+
+        """
+        return [
```

Comment:
I verified we didn't drop any rules between this list and the old `DEFAULT_RULES` list.